Tests/Recurly/Account_Test.php

```diff
@@ -201,4 +201,42 @@ public function testCreateWithBillingInfoToken() {
 
     $this->assertInstanceOf('Recurly_Account', $account);
   }
+
+  /**
+   * Tests regarding account hierarchy.
+   */
+  public function testAccountHierarchy() {
+    $this->client->addResponse('POST', '/accounts', 'accounts/hierarchy/create-201.xml');
```

Comment:
Is there a reason we need to create new fixtures for all these? Could we possibly re-use fixtures? I'm open to either but I find adding fixtures to be cumbersome since they are managed by hand. They tend to go stale quick.

Reply:
I'm okay with it 👍